server/lib/payments.js

```diff
@@ -389,7 +390,7 @@ const validatePayment = payment => {
     throw new Error('Interval should be null, month or year.');
   }
 
-  if (!payment.amount) {
+  if (!payment.amount && !payment.platformTip) {
```

Comment:
@kewitz : The `platformTip` might not be populated on every case; but this is done so that after the `payment.amount` is paid the user adds the platform tip (in the second modal) and in that case the `payment.amount` is will be null or undefined whereas we do send the platform tip. So the payment has to go through with only the `platformTip`. 😄